apps/website-25/src/pages/api/courses/[courseSlug]/[unitId].ts

```diff
@@ -28,9 +35,16 @@ export default makeApiRoute({
     throw new createHttpError.NotFound('Unit not found');
   }
 
+  const unitExercises = (await db.scan(exerciseTable, {
```

Comment:
Rather than affiliating exercises with a unit like this, I think we should embed them as components within the unit content itself.

E.g. the `unit.content` markdown has something like `<MCQExercise id="rec1234" />`